src/librustc/Cargo.toml

```diff
@@ -15,7 +15,7 @@ fmt_macros = { path = "../libfmt_macros" }
 graphviz = { path = "../libgraphviz" }
 jobserver = "0.1"
 log = "0.3"
-owning_ref = "0.3.3"
+owning_ref = { git = "https://github.com/Zoxc/owning-ref-rs.git" }
```

Comment:
the rustc build process shouldn't depend on random git repos - that's just too fragile. We want to keep things building in N years.